modules/ROOT/pages/monitoring/metrics/reference.adoc

```diff
@@ -92,10 +92,18 @@ By default, database metrics include:
 |<prefix>.bolt.response_success|(unsupported feature) When `internal.server.bolt.response_metrics` is enabled, number of `encounteredsuccess` responses. (counter)
 |<prefix>.bolt.response_ignored|(unsupported feature) When `internal.server.bolt.response_metrics` is enabled, number of `encounteredignored` responses (counter)
 |<prefix>.bolt.response_failed|(unsupported feature) When `internal.server.bolt.response_metrics` is enabled, number of `encounteredinstances` of a given error code. (counter)
-|<prefix>.bolt_driver.managed_transaction_function_calls|The total number of managed transaction function calls. (counter)
-|<prefix>.bolt_driver.unmanaged_transaction_calls|The total number of un-managed transaction function calls. (counter)
-|<prefix>.bolt_driver.implicit_transaction_calls|The total number of implicit transaction function calls. (counter)
-|<prefix>.bolt_driver.execute_calls|The total number of driver-level execute function calls. (counter)
+|===
+
+[[bolt-driver-metrics]]
+.Bolt Driver metrics
```

Comment:
```suggestion
.Bolt driver metrics
```

Reply:
I wonder if we could have labels in the description. If that's not possible, we can add it above the table as we do it for the configuration settings.

Reply:
'Driver` starts with capital letter in the code. I'd prefer to keep spelling as it's in the code, we can fix it later.

Reply:
I like the idea about labels, it would be much better to have one label instead of four.